packages/table/src/table.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import {TableBody} from './table-body'
+import {TableCell} from './table-cell'
+import {TableHead} from './table-head'
+import {TableHeaderCell} from './table-header-cell'
+import {TableRow} from './table-row'
+import {TextTableCell} from './text-table-cell'
+import {TextTableHeaderCell} from './text-table-header-cell'
+import {SearchHeader} from './select-table-header-cell'
+import styled from 'styled-components'
+
+export default class Table extends React.Component {
```

Comment:
export `styled()` component so it can be further styled. See `packages/spinner/src/index.tsx` line 94 for reference.